src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -341,7 +341,7 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
       this._loadUserConfig();
       this._initUserConfigAutoSave();
     }
-    if (!this.userConfig.limit) {
+    if (this.limit !== 10 || !this.userConfig.limit) {
```

Comment:
Why are we hard-coding a `10` here? If this is a default, we should probably use a constant.

Reply:
created a constant for it. It was the default value for the list of entries to show in a table!